pallets/automation-price/src/lib.rs

```diff
@@ -563,8 +563,8 @@ pub mod pallet {
 			// TODO: use sudo and remove this feature flag
 			// TODO: needs fees if opened up to non-sudo
 			// When enable dev-queue, we skip this check
-			#[cfg(not(feature = "dev-queue"))]
-			ensure_root(origin)?;
+			// #[cfg(not(feature = "dev-queue"))]
+			// ensure_root(origin)?;
```

Comment:
Yeah this line is not really needed, because we always have sudo access with `dev-queue` in local environment.

Reply:
okay, yeah I remember commenting on this line 😅